cowprotocol/accounting/rewards/excluded_batches_query_3490353.sql

```diff
@@ -168,3 +169,62 @@ where
     or tx_hash = 0x7a56ca3e7f2a6a41f829fed953c321010a3ec3c655909f382bff676d39e18dd0
     or tx_hash = 0x46c5c6768774b92af828fb18be81e88baa28d28292f7c2d048d8c3080ea9da5c -- involving token that basically couldn't trade afterwards
     or tx_hash = 0x4b2abb45491995674442a041a82e94a863428bca4f4c956805f8ec9f802c9fba
+
+    -- for week of Dec 17 - Dec 24, 2024
+    or tx_hash = 0xF7A16B761CB5B09F68350607675AF07CE48DC772D4FF1C6E9EDA5E4E3F62A139
+    or tx_hash = 0xD5295F40E2685C5FD4DE432C2D377F9D5E71877C187FAE5D0FC44A5ED66FC142 -- involving token that basically couldn't trade afterwards
+    or tx_hash = 0x6C43B1A12293D014CEC6F77078D7947A85AFD849F159356E56E8FC336CDB6A7C
+
+    -- for week of Dec 31, 2024 - Jan 7, 2025
+    or tx_hash = 0xd1ca15a5921781979d4a156da233668940d580cf5e04a721f874aeae7c4748e4
+
+    -- for week of Jan 7 - Jan 14, 2025
+    or tx_hash = 0x132BDC1984EA83665DD590FA5C98C82752426EA060E32CA2151E38643C6EFBD6
+    or tx_hash = 0xBBFFB1584A20BDFF8ABFABFD66C773EE3BEF16E7BF9549FFDFA88781BAE347EB
+
+    -- for week of Jan 14 - Jan 21, 2025 on mainnet
+    or tx_hash = 0x8254AE2FC56214657139B0A1E934D00937019BD8B6F873641A34F1EDBE0FBC91
+    or tx_hash = 0xA8C8F763A7039FD76ADA9067C5868A2A249C1B00C1B30533DFF73B08DDF309D7
+    or tx_hash = 0xBD232581323F79D6A5E31D3F42209B3769E1CBA010469948FB7D3D3F0E5C8060
+    or tx_hash = 0xA9813D70A87C609F76111FC74C52D820D2FFF2BA68F179C0A339387034555EF8
+
+    -- for week of Jan 21 - Jan 28, 2025 on mainnet
+    or tx_hash = 0xB99EA4B15A54EE68D02D695C5CDD1F60CB33D2F2D50DA5BCA277C8ADAE108208
+    or tx_hash = 0x74e730407d3061e60527629123c1e7b43e3e73b4ba4232bd077c7109ac5e8325
+    or tx_hash = 0x08afb56d779bc2034bfcc78a1d470cc82c8c55c4394fda3875d426de869f8ca2
+
+    -- for week of Feb 4 - Feb 11, 2025 on mainnet
+    or tx_hash = 0xf5a574a1bac9d5ffabeb9c959af3018a41b1570cbf4a0ddf84e9ddda83dac6df
+    or tx_hash = 0x069B2C966D9C95119983EA51995D642F320AE3B8252EE9AF643AFEB6D7CA3D69
+    or tx_hash = 0x588397A4C1D4671E65A3D8355F3E7C9CE2DC6D6013E06BB3332C3357EE284BA8
+    or tx_hash = 0xCA7672EF67787948743F5D2C6C56DB8946488BFEFD0566F9F91A7F91F66EF2A3
+    or tx_hash = 0x1082AD668F906AC04231296C427BF57E6589E3A907F3F876752413FC3B98420D
+
+    -- for week of Feb 11 - Feb 18, 2025 on mainnet
+    or tx_hash = 0x33955F26DCFA23CF4C20C0ECA1C914C140C01397C0E521ADF494B71F871B9F7F
+    or tx_hash = 0x50D5F477B7F62BF2F4CB258E9D5AC375033DF56F4C0CC07DE4E9C585D90AA390
+
+    -- for week of Feb 25 - March 4, 2025 on mainnet
+    or tx_hash = 0x34AD176862C6D51BDD8232B67BF4C689CF0707CEC18E17E9731017A1F3823F6A
+
+    -- for week of March 4 - March 11, 2025 on mainnet
+    or tx_hash = 0x1F14B47CCA77FBA18F6AD87372C9F942617FAF9C787622BBD8F56D187EF069FA
+
+-- Base
+union all
```

Comment:
The line on Dune is
```suggestion
union
```
which is a `union distinct`.
It should not make a difference but also does not look like an error from copy-pasting.